profile/README.md

```diff
@@ -10,7 +10,7 @@
 <a href="https://github.com/open-horizon?tab=repositories"><img src="https://badges.pufler.dev/repos/open-horizon" alt="Open Horizon" /> </a>
 <img src="https://img.shields.io/github/stars/open-horizon?label=Stars" alt="Total Stars">
 <!--
-<a href="https://twitter.com/lf_edge)" alt="twitter"><img src="https://img.shields.io/badge/twitter-@lf_edge--twitter-yellow.svg" alt="twitter"></img></a>
+<a href="https://twitter.com/lf_edge" alt="twitter"><img src="https://img.shields.io/badge/twitter-@lf_edge--twitter-yellow.svg" alt="twitter"></img></a>
```

Comment:
Please remove closing `</img>` tag.  Image is not an HTML container, so it does not have a closing tag in HTML 4.x and HTML 5.